backend/clamav/IAvService.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Pims.Av
+{
+    public interface IAvService
+    {
+        public Task Scan(IFormFile file);
```

Comment:
In theory this could support another av implementation besides clam, but currently the only service that implements this is clam.